packages/calling/src/CallHistory/callHistoryFixtures.ts

```diff
@@ -337,3 +337,16 @@ export const MOCK_SESSION_EVENT: CallSessionEvent = {
   timestamp: 12345,
   trackingId: 'tracking-id',
 };
+
+export const MOCK_LOCUS_SESSION_EVENT: CallSessionEvent = {
+  id: 'id',
+  data: {
+    userSessions: {
+      userSessions: [WEBEX_CALL_SESSION],
```

Comment:
could we use a different object that matches the legacy session type?

Reply:
Yeah I am yet to fetch the object from the logs, will update